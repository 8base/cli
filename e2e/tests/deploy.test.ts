import { runner } from '../runner';

it('As a user, I can use help flag for see help information for `deploy` command.', async () => {
  const { stdout } = await runner()('deploy', '--help');

  expect(stdout).toMatchSnapshot();
});