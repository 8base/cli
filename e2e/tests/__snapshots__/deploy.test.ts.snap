// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`As a user, I can use help flag for see help information for \`deploy\` command. 1`] = `"8base deploy [OPTIONS]"`;
